context/app/api/vitessce_confs/assay_confs.py

```diff
@@ -105,6 +109,10 @@ class CytokitSPRMViewConfigError(Exception):
 
 
 class TiledSPRMViewConfBuilder(ViewConfBuilder):
+    """Wrapper class for generating many "first generation"
```

Comment:
Do we have an approximate date for the last dataset of this form? Maybe comment on what replaces it?

Reply:
I can try to add some dates to these

Reply:
Hmm, there's no date filter in the UI so it's hard.  I'll try to ballpark it but also link to the cwl.